### PR TITLE
ticdc: add more faq

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Add more FAQ about TiCDC.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### CODE REVIEW

File: ticdc/troubleshoot-ticdc.md [hunk 1]
```diff
@@ -110,6 +110,64 @@ cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/" --tz=Asia/Sh
...
+* `3`: 任务已删除，接口请求后会结束所有同步 `processor`，并清理同步任务配置信息。同步状态保留，只提供查询，没有其他实际功能。
+
+## 为什么 TiCDC 到 Kafka 的同步任务延时越来越大？
+
+* 请参考 [如何查看 TiCDC 同步任务是否正常？](/ticdc/troubleshoot-ticdc.md#如何查看-TiCDC-同步任务是否正常) 检查下同步任务的状态是否正常。
+* 请适当调整 Kafka 的以下参数：
+  * message.max.bytes
+  * replica.fetch.max.bytes
+  * fetch.message.max.bytes
+  * socket.request.max.bytes
```
Comment (overvenus) [inline ticdc/troubleshoot-ticdc.md:173]: PTAL, how to tweak these params?

File: ticdc/troubleshoot-ticdc.md [hunk 2]
```diff
@@ -110,6 +110,78 @@ cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/" --tz=Asia/Sh
...
+上述命令的输出中 `admin-job-type` 标志这个同步的任务的状态：
+
+* `0`: 状态正常。
+* `1`: 任务暂停，停止任务后所有同步 `processor` 会结束退出，同步任务的配置和同步状态都会保留，可以从 `checkpoint-ts` 恢复任务。
+* `2`: 任务恢复，同步任务从 `checkpoint-ts` 继续同步。
+* `3`: 任务已删除，接口请求后会结束所有同步 `processor`，并清理同步任务配置信息。同步状态保留，只提供查询，没有其他实际功能。
+
+## 为什么 TiCDC 到 Kafka 的同步任务延时越来越大？
+
+* 请参考 [如何查看 TiCDC 同步任务是否正常？](/ticdc/troubleshoot-ticdc.md##如何查看-ticdc-同步任务是否正常) 检查下同步任务的状态是否正常。
```
Comment (TomShawn) [inline ticdc/troubleshoot-ticdc.md:168]:
```suggestion
* 请参考 [如何查看 TiCDC 同步任务是否正常？](/ticdc/troubleshoot-ticdc.md#如何查看-ticdc-同步任务是否正常) 检查下同步任务的状态是否正常。
```

File: ticdc/troubleshoot-ticdc.md [hunk 3]
```diff
@@ -110,6 +110,78 @@ cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/" --tz=Asia/Sh
...
+
+可以使用 cdc cli 查询同步任务的状态。比如：
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed query --pd=http://10.0.10.25:2379 --changefeed-id 28c43ffc-2316-4f4f-a70b-d1a7c59ba79f
+```
+
+上述命令的输出中 `admin-job-type` 标志这个同步的任务的状态：
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:159]: 不推荐用户看这个状态，建议使用最新的 cdc cli changefeed list / cdc cli changefeed query --simple 查看
ref: https://github.com/pingcap/ticdc/pull/721

File: ticdc/troubleshoot-ticdc.md [hunk 4]
```diff
@@ -138,13 +138,28 @@ cdc cli changefeed create --pd=http://10.0.10.25:2379 --sink-uri="kafka://127.0.
...
-上述命令的输出中：
+上述命令输出如下：
 
-* `replication_gap` 是 TiCDC 已经同步到下游的数据于上游的数据差了多少时间。
-* `sink_gap` 是 TiCDC 从上游获取的数据和最新的数据差了多少时间。
+```json
+[{
+    "id": "4e24dde6-53c1-40b6-badf-63620e4940dc",
+    "summary": {
+      "state": "normal",
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:150]: 可以增加一下对 state 的解释
normal: 正常同步
stopped: 停止同步（手动暂停或出错）
removed: 已删除任务

File: ticdc/troubleshoot-ticdc.md [hunk 5]
```diff
@@ -110,6 +110,119 @@ cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/" --tz=Asia/Sh
...
+* `2`: 任务恢复，同步任务从 `checkpoint-ts` 继续同步。
+* `3`: 任务已删除，接口请求后会结束所有同步 `processor`，并清理同步任务配置信息。同步状态保留，只提供查询，没有其他实际功能。
+
+## 为什么 TiCDC 到 Kafka 的同步任务延时越来越大？
+
+* 请参考 [如何查看 TiCDC 同步任务是否正常？](/ticdc/troubleshoot-ticdc.md#如何查看-ticdc-同步任务是否正常) 检查下同步任务的状态是否正常。
+* 请适当调整 Kafka 的以下参数：
+    * `message.max.bytes`，将 Kafka 的 server.properties 中该参数调大到 33554432 （32 MB）。
+    * `replica.fetch.max.bytes`，将 Kafka 的 server.properties 中该参数调大到 33554432 （32 MB）。
+    * `fetch.message.max.bytes`，适当调大 comsumer.properties 中该参数，确保大于 `message.max.bytes`。
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:187]: 这 3 个值都建议大一点，可以调到 1G (1073741824)

File: ticdc/troubleshoot-ticdc.md [hunk 6]
```diff
@@ -110,6 +110,119 @@ cdc cli changefeed create --sink-uri="mysql://root@127.0.0.1:3306/" --tz=Asia/Sh
...
+
+## TiCDC 把数据同步到 Kafka 时，是把一个事务内的所有变更都写到一个消息中吗？如果不是，是根据什么划分的？
+
+不是，根据配置的分发策略不同，有不同的划分方式，包括 default, row id, table, ts，更多请参考[同步任务配置文件描述](/ticdc/manage-ticdc.md#同步任务配置文件描述)。
+
+## TiCDC 把数据同步到 Kafka 时，能在 TiDB 中控制单条消息大小的上限吗？
+
+不能，目前 TiCDC 控制了消息大小 4MB
```
Comment (amyangfei) [inline ticdc/troubleshoot-ticdc.md:195]: 4 MB 是单个 key-value 消息的最大大小；实际每次向 kafka broker 发送的批量消息最大是 512MB

**Comment by TomShawn** [comment]
@amyangfei PTAL again

**Pull request merged**